src/main.ts

```diff
@@ -541,7 +541,7 @@ function generateDecode(
   func = func
     .addStatement('const reader = input instanceof Uint8Array ? new Reader(input) : input')
     .addStatement('let end = length === undefined ? reader.len : reader.pos + length')
-    .addStatement('const message = Object.create(base%L) as %L', fullName, fullName);
+    .addStatement('const message = {...base%L} as %L', fullName, fullName);
```

Comment:
This is pretty minor, but can you change this to `{ ...base%L }` i.e. add parens after/before the parens. That is what prettier will eventually make this when we have prettier formatted output.

Reply:
Fixed 👍 . Please squash this when you merge. Some bash script changes snuck into my commits